components/eamxx/cmake/machine-files/aurora.cmake

```diff
@@ -11,15 +11,6 @@ set(EKAT_MPI_THREAD_FLAG "-d" CACHE STRING "")
 SET(SYCL_COMPILE_FLAGS "-std=c++17 -fsycl -fsycl-device-code-split=per_kernel -fno-sycl-id-queries-fit-in-int -fsycl-unnamed-lambda")
 SET(SYCL_LINK_FLAGS "-fsycl -fsycl-link-huge-device-code -fsycl-device-code-split=per_kernel  -fsycl-targets=spir64_gen -Xsycl-target-backend \"-device 12.60.7\"")
 
-set(CMAKE_CXX_FLAGS " -\-intel -Xclang -fsycl-allow-virtual-functions -mlong-double-64 -O3 -DNDEBUG ${SYCL_COMPILE_FLAGS}" CACHE STRING "" FORCE)
-set(CMAKE_Fortran_FLAGS "-fc=ifx -O3 -DNDEBUG -DCPRINTEL -g" CACHE STRING "" FORCE)
-set(CMAKE_C_FLAGS "-O3 -DNDEBUG" CACHE STRING "" FORCE)
+set(CMAKE_CXX_FLAGS " -\-intel -Xclang -fsycl-allow-virtual-functions -mlong-double-64 ${SYCL_COMPILE_FLAGS}" CACHE STRING "" FORCE)
+set(CMAKE_Fortran_FLAGS "-fc=ifx -DCPRINTEL -g" CACHE STRING "" FORCE)
 set(CMAKE_EXE_LINKER_FLAGS " -lifcore -\-intel -Xclang -fsycl-allow-virtual-functions -lsycl -mlong-double-64 -DNDEBUG ${SYCL_LINK_FLAGS} -fortlib" CACHE STRING "" FORCE)
```

Comment:
Probably good to remove `-DNDEBUG` here too.

Reply:
I followed both of your suggestions.